jest.config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "preset": "ts-jest",
+  "testEnvironment": "node",
+  "globals": {
+    "ts-jest": {
+      "tsConfig": "./tsconfig.json"
+    }
+  },
+  "moduleDirectories": ["node_modules", "src"],
+  "moduleFileExtensions": ["ts", "tsx", "html", "js", "json"],
+  "moduleNameMapper": {
+    "^@core/(.*)": "<rootDir>/src/app/$1",
+    "^@assets/(.*)": "<rootDir>/src/assets/$1",
+    "^@environments/(.*)": "<rootDir>/src/environments/$1"
+  },
+  "setupFiles": ["<rootDir>/test-setup.js"]
```

Comment:
please take the latest `jest.config.json` from CRA repository- it has updated aliases and config

Reply:
Not user if jest will work with react native out fo the box... with teh same config, but lets left unit tests fo native as a next thing not part of this story.